docs/01-Chart-Configuration.md

```diff
@@ -257,6 +257,18 @@ afterBody | `Array[tooltipItem], data` | Text to render after the body section
 beforeFooter | `Array[tooltipItem], data` | Text to render before the footer section
 footer | `Array[tooltipItem], data` | Text to render as the footer
 afterFooter | `Array[tooltipItem], data` | Text to render after the footer section
+dataPoints | `Array[CustomTooltipDataPoint], data` | List of matching point informations.
+
+#### CustomTooltipDataPoint
```

Comment:
Hello @etimberg ,

I am not sure what you want me to do here. Could you explain more ?

Which sections do you want to merge ? If you are talking about `CustomTooltipDataPoint` and the one just before, `dataPoints` is an array so it should be defined as an external section right ?

Also for `x` and `y`, which code snippet are you talking about ?